features/your_results_frequency_selector.feature

```diff
@@ -1,36 +1,69 @@
 Feature:
-  As an employee, I want to be able to see my results in a frequency different to how I input my salary because I want to understand how the contributions will impact my monthly or weekly take-home pay.
+  In order to understand how the contributions will impact my monthly or weekly take-home pay
+  As an employee
+  I want to be able to see my results in a frequency different to how I input my salary
 
   Background:
     Given I am on the Your Details step
+    And I am a "25" year old "female"
```

Comment:
@Tr4pSt3R but it was @willhall88 suggestion to use Given I am 25 years old male step. And now we are having this discussion on what should be use. Something is not right on this discussion, including on this PR.